docker/Dockerfile

```diff
@@ -0,0 +1,77 @@
+FROM nvidia/cuda:11.8.0-cudnn8-runtime-ubuntu22.04
+
+ARG DEFAULT_UID=1000
+ARG DEFAULT_GID=1000
+ENV DEFAULT_UID $DEFAULT_UID
+ENV DEFAULT_GID $DEFAULT_GID
+ENV PUSER "rooper"
+ENV PGROUP "rooper"
+ENV PUSER_PRIV_DROP true
+ENV PUSER_RLIMIT_UNLOCK true
+
+ENV DEBIAN_FRONTEND noninteractive
+ENV PYTHONUNBUFFERED 1
+ENV PYTHONDONTWRITEBYTECODE 1
+
+ENV BUFFALO_L_URL "https://github.com/deepinsight/insightface/releases/download/v0.7/buffalo_l.zip"
+# ENV INSWAPPER_128_URL "https://1drv.ms/u/s!AsHA3Xbnj6uAgxhb_tmQ7egHACOR?e=CPoThO"
+
+COPY --chmod=644 ./inswapper_128.onnx /roop/inswapper_128.onnx
+COPY ./requirements.txt /tmp/requirements.txt
```

Comment:
Why not checking out the repo here instead? It is not a good idea to create a container that only works within a working copy of the project